dev-ruby/async-io/files/async-io-1.34.3-tests.patch

```diff
@@ -0,0 +1,22 @@
+From fe6f1972c74ec0c5107e127cdb299f41b798e3dd Mon Sep 17 00:00:00 2001
```

Comment:
Could you just add bug reference here?

Reply:
I've fixed this up to get this in. Thanks!

Reply:
> I've fixed this up to get this in. Thanks!

I just got back, thank you for doing it for me!